src/app.jsx

```diff
@@ -101,7 +106,37 @@ export const App = () => {
                 .finally(() => setLoadingResources(false));
     }, []);
 
-    if ((superuser.allowed && systemSocketInactive) || loadingResources) {
+    useEffect(() => {
+        async function checkVirtualization() {
+            const hardwareVirtCheck = await cockpit.script("virt-host-validate | grep 'Checking for hardware virtualization'");
+
+            setVirtualizationEnabled(hardwareVirtCheck.includes('PASS'));
+        }
+
+        checkVirtualization();
+    }, []);
+
+    if (!virtualizationEnabled && !emptyStateIgnored) {
+        return (
+            <EmptyState>
+                <EmptyStateIcon icon={WrenchIcon} />
+                <Title headingLevel="h4" size="lg">
+                    {_("Virtualization support is not available")}
+                </Title>
+                <EmptyStateBody>
+                    <Text>{_("Enable virtualization support in BIOS/EFI settings.")}</Text>
+                    <Text>
+                        {_(`Instructions vary per manufacturer, but usually involves pressing a hot key such as ESC, F1, F12, or Del during boot.
+                        In the settings screen, look for "virtualization", "VM", "VMX', "SVM", "VTX", "VTD". It will be different on every computer.
+                        Enable any of these options. Consult your computer's manual for details.`)}
```

Comment:
```suggestion
                        {_(`Changing BIOS/EFI settings is specific to each manufacturer. It involves pressing a hotkey during boot (ESC, F1, F12, Del).
                        Enable a setting called "virtualization", "VM", "VMX', "SVM", "VTX", "VTD".
                        Consult your computer's manual for details.`)}
```

I've tried to trim it down a bit. How's this?